images/cilium/Dockerfile

```diff
@@ -10,14 +10,14 @@ ARG NOSTRIP
 ARG LOCKDEBUG
```

Comment:
commit can be squashed

Reply:
I'll probably squash the whole lot at the end into one or two, still WIP.